modules/pensions/app/models/pensions/saved_claim.rb

```diff
@@ -85,6 +87,38 @@ def send_confirmation_email
           'confirmation_number' => guid
         }
       )
+
+      insert_notification
+    end
+
+    ##
+    # insert notifcation after VANotify email send
+    #
+    # @see ClaimVANotification
+    #
+    def insert_notification
+      ClaimVANotification.create!(
+        form_type: form_id,
+        saved_claim_id: id,
+        email_sent: true,
+        email_template_id: Settings.vanotify.services.va_gov.template_id.form527ez_confirmation_email,
+        created_at: DateTime.now,
+        updated_at: DateTime.now
+      )
+    end
+
+    ##
+    # Find notifcation by args*
+    #
+    # @param email_template_id
+    # @see ClaimVANotification
+    #
+    def va_notification?(email_template_id)
+      ClaimVANotification.find_by(
```

Comment:
a relationship to claim_va_notifications (for example) would simplify this and all `includes` on the module query